src/excmds.ts

```diff
@@ -166,19 +164,15 @@ export async function getrss(url: string, type?: string, ...title: string[]) {
                 type = links[0].type
                 break
             default:
-                return fillcmdline("getrss")
+                return fillcmdline("rssexec")
         }
     }
-    let rsscmd = config.get("rsscmd")
-    if (rsscmd == "") {
-        return setclip(url)
-    }
-    rsscmd = rsscmd
+    let rsscmd = config
+        .get("rsscmd")
         .replace("%u", url)
```

Comment:
It seems reasonable to add the URL to the end of the command like we do for searchurls if no %u is found.

I think it probably makes sense to add a space before it too (but not before %u).